### DISCUSSION TITLE
! There were one or more problems detected. Please review the errors and correct them and run uno-check again.

### DISCUSSION THREAD

**Discussion by GGDimitrov21** [unoplatform/uno#19079] [Q&A] [answered]
Hi, I did reinstall uno.check and have .NET 9 installed but whenever I run uno-check and goes through everything flawlessly, at the end it says: "! There were one or more problems detected.
Please review the errors and correct them and run uno-check again." Do you know how to fix it or at least how and where to review the errors?

**Comment by DevTKSS** [comment, marked as answer]
yes, lookup the troubleshooting:

https://platform.uno/docs/articles/external/uno.check/doc/using-uno-check.html?tabs=windows#troubleshooting

had same issue, done at last step the -verbose and seen what its problem was
